internal/manager/manager.go

```diff
@@ -633,6 +632,22 @@ func (s *Manager) Setup(ctx context.Context, input SetupInput) error {
 		s.Config.Set(config.Generated, input.GeneratedLocation)
 	}
 
+	// if blobs path was provided then use filesystem based blob storage
+	if input.BlobsLocation != "" {
+		if !c.HasOverride(config.BlobsPath) {
+			if exists, _ := fsutil.DirExists(input.BlobsLocation); !exists {
+				if err := os.Mkdir(input.BlobsLocation, 0755); err != nil {
```

Comment:
possible enhancement would be to use os.MkdirAll to create all needed dirs in path and not just the final one? ( probably the same can be applied for the generated and configdir as well)